utils/epinow2/constants.py

```diff
@@ -0,0 +1,75 @@
+shared_params = {
+    "seed": 42,
+    "horizon": 14,
+    "priors": {"rt": {"mean": 1.0, "sd": 0.2}, "gp": {"alpha_sd": 0.01}},
+    "parameters": {
+        "path": "data/parameters.parquet",
+        "blob_storage_container": None,
+    },
+    "sampler_opts": {
+        "cores": 4,
+        "chains": 4,
+        "adapt_delta": 0.99,
+        "max_treedepth": 12,
+    },
+}
+
+states = [
```

Comment:
For the new version of the pipeline, I believe we decided to use upper case for the state abbreviations. Is that correct @zsusswein